src/Workspaces/CSharp/Portable/LanguageServices/CSharpTypeInferenceService.TypeInferrer.cs

```diff
@@ -1903,24 +1904,6 @@ private IEnumerable<TypeInferenceInfo> InferTypeInYieldStatement(YieldStatementS
                     : SpecializedCollections.EmptyEnumerable<TypeInferenceInfo>();
             }
 
-            private static ITypeSymbol GetMemberType(ISymbol memberSymbol, out bool isAsync)
```

Comment:
This was equivalent to our existing symbol extension by the same name other than it also did the isAsync thing. With pattern matching that's now easy to inline that one bit.

Reply:
nice!